apps/aeutils/src/aeu_requests.erl

```diff
@@ -143,10 +143,13 @@ get_header_by_hash(Uri, Hash) ->
             {error, unexpected_response}
     end.
 
--spec get_n_hashes(http_uri_uri(),  binary(), non_neg_integer()) -> response([{integer(), binary()}]).
-get_n_hashes(Uri, Hash, N) when is_integer(N) ->
+%% Get the next n hashes and their heights
+-spec get_n_successors(http_uri_uri(),  binary(), non_neg_integer()) -> response([{integer(), binary()}]).
+get_n_successors(Uri, Hash, N) when is_integer(N) ->
     EncHash = aec_base58c:encode(block_hash, Hash),
-    Response = process_request(Uri, 'GetHeadersByHash', [{"hash", EncHash}, {"number", integer_to_list(N)}]),
+    Response = process_request(Uri, 'GetHeadersByHash', [{"hash", EncHash}, 
+                                                         {"number", integer_to_list(N+1)}, 
+                                                         {"direction", forward}]),
     case Response of
```

Comment:
To follow the spec (and please Dialyzer) you should change:
`{"direction", forward}` -> `{"direction", "forward"}`

Reply:
Thanks for pointing this out... took me also a while to find this